sandbox/basic.py

```diff
@@ -139,7 +141,8 @@ def key_x(self):
         self.panic(self.tree)
 
 
-app = BasicApp(css_file="basic.css", watch_css=True, log="textual.log")
+css_file = Path(__file__).parent / "basic.css"
```

Comment:
Just using an absolute path rather than a relative one here, so this App can be used from any other folder :-)